### DISCUSSION TITLE
打包后部分样式错乱

### DISCUSSION THREAD

**Discussion by xqGosky** [NervJS/taro#11200] [Q & A]
我两台电脑 其中电脑1 预览跟打包都没有问题  另一台 电脑2 则是打包后部分样式会消失或者错乱  有些方法也会异常 例如一些点击事件会失效,但是电脑2是我一直在用的电脑 之前一直都没有问题  大概有两三个月没用了 今天用突然发现不行了  taro-cli也重新安装过了 不行

**Comment by Dante-dan** [comment]
盲猜依赖没锁，某个依赖小版本更新了。对比两个 lock 文件就知道了
